src/ages/age-one/getUserUnclaimedTokens.ts

```diff
@@ -24,23 +24,38 @@ export const getUserUnclaimedTokensFromDistribution = async (
     blockNumber,
   );
 
-  return userBalancesToUnclaimedTokens(userBalances.balances, endDate, epoch);
+  return userBalancesToUnclaimedTokens(address, userBalances.balances, endDate, epoch);
 };
 export const userBalancesToUnclaimedTokens = (
+  userAddress: string,
   balances: UserBalance[],
   endDate: BigNumber,
   epoch: keyof typeof configuration.epochs,
-) =>
-  balances
+) => {
+  let prevEpochRewards = BigNumber.from(0);
+
+  // add the previous epoch rewards to sum rewards for root computation
+  if (epoch === "epoch2") {
+    const strRewards = epochOneResult.distribution.find(
+      (d) => d.address.toLowerCase() === userAddress.toLowerCase(),
+    )?.unclaimedRewards;
```

Comment:
```suggestion
    )?.accruedRewards;
```

Reply:
This is misleading: we're not summing up unclaimed rewards but accrued rewards since the beginning (previous epoch only for now, but will get refactored later)
This is important because otherwise someone could claim rewards between from the moment the root hash is computed up until the root hash is published to the RewardsManager, and claim again after the root hash has been published

Nothing critical though: the RewardsManager keeps track of claimed rewards and thus it would lead to a revert (still, it's a frontend bug)